SeleniumCookies/wrapper.py

```diff
@@ -329,7 +362,8 @@ def __add_session_cookies_lz4(self, cj):
             file_obj.read(8)
             json_data = json.loads(lz4.block.decompress(file_obj.read()))
         except ValueError as e:
-            logging.error(f'Error parsing firefox session JSON LZ4: str(e)')
+            logging.error("{'ErrorCode':'%s' ,'Error':'%s'}",

```

Comment:
You can abstract away the map dereference from here

Reply:
Ok, but any reason for this?

Reply:
I didn't get the point here :?

Reply:
@L04DB4L4NC3R

Reply:
The mean reason is redundancy. Ideally we should only need to pass the error code and the corresponding message should get logged on it's own.

Reply:
The mean reason is redundancy. Ideally we should only need to pass the error code and the corresponding message should get logged on it's own.

Reply:
The mean reason is redundancy. Ideally we should only need to pass the error code and the corresponding message should get logged on it's own.